lib/matplotlib/lines.py

```diff
@@ -1405,34 +1404,61 @@ class _AxLine(Line2D):
     transform at draw time.
     """
 
+    def __init__(self, xy1, xy2, slope, **kwargs):
+        super().__init__([0, 1], [0, 1], **kwargs)
+
+        if (xy2 is None and slope is None or
+                xy2 is not None and slope is not None):
+            raise TypeError(
+                "Exactly one of 'xy2' and 'slope' must be given")
+
+        self._slope = slope
+        self._xy1 = xy1
+        self._xy2 = xy2
+        self._transform = kwargs.get("transform")
```

Comment:
I think this would have already been set by super `__init__`; can this be dropped?

Reply:
Indeed, it seems to work without this - I just didn't know if I should rely on this in the subclass (as I use `self._transform` directly in the overridden `get_transform` function) instead of setting it explicitly. Changed in 21deaf4.